src/modules/luksbootkeyfile/LuksBootKeyFileJob.cpp

```diff
@@ -172,6 +172,22 @@ hasUnencryptedSeparateBoot()
     return false;
 }
 
+static bool
+hasEncryptedRoot()
```

Comment:
I see you've just copy-pasted an existing function; I'm going to Have Opinions about that, but only after we've merged your work, then I'll hit it with a more-modern-C++ hammer.